fuzz/smime.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * https://www.openssl.org/source/license.html
+ * or in the file LICENSE in the source distribution.
+ */
+
+#include "fuzzer.h"
+#include <openssl/err.h>
+#include <openssl/pkcs7.h>
+#include <openssl/x509.h>
+#include <stdio.h>
+
+int FuzzerInitialize(int *argc, char ***argv) { return 1; }
```

Comment:
Nit: For functions only we always place the opening `{` on the next line. Here and in other functions below. Also the "return 1;" should be a line on its own (not on same line as `{}`)